Enhance product details documentation with scoped contexts support (#432)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/initialization.mdx

});
```

## Scope Configuration

> The `scope` option enables **scoped contexts** for the PDP drop-in component. When a scope is provided, the component operates within an isolated context, allowing multiple independent PDP instances to run on the same page without conflicts.

Reviewer: ```suggestion
The `scope` option enables **scoped contexts** for the PDP drop-in. When provided, the component runs in an isolated context. This allows multiple independent PDP instances on the same page without conflicts.
```
